mlos_bench/mlos_bench/storage/sql/schema.py

```diff
@@ -272,11 +275,36 @@ def meta(self) -> MetaData:
         """Return the SQLAlchemy MetaData object."""
         return self._meta
 
+    def _get_alembic_cfg(self, conn: Connection) -> config.Config:  # pylint: disable=no-self-use
+        alembic_cfg = config.Config(
+            path_join(str(files("mlos_bench.storage.sql")), "alembic.ini", abs_path=True)
+        )
+        alembic_cfg.attributes["connection"] = conn
+        return alembic_cfg
+
     def create(self) -> "DbSchema":
         """Create the DB schema."""
         _LOG.info("Create the DB schema")
         assert self._engine
         self._meta.create_all(self._engine)
+        with self._engine.connect() as conn:
```

Comment:
Use `.begin()` for committable transactions:

```suggestion
        with self._engine.begin() as conn:
```